dali/operators/reader/loader/video/frames_decoder.cc

```diff
@@ -19,15 +19,57 @@
 
 
 namespace dali {
+int MemoryVideoFile::Read(unsigned char *buffer, int buffer_size) {
+  int left_in_file = size_ - position_;
+  if (left_in_file == 0) {
+    return AVERROR_EOF;
+  }
+
+  int to_read = std::min(left_in_file, buffer_size);
+  std::copy(data_ + position_, data_ + position_ + to_read, buffer);
+  position_ += to_read;
+  return to_read;
+}
+
+int64_t MemoryVideoFile::Seek(int64_t new_position, int origin) {
```

Comment:
Why is this arg names `origin`. As I understand it's a seeking method.

Reply:
Changed it to `mode`. Yes, this is the seeking method.